### ISSUE TITLE
a question about slot

### ISSUE THREAD

**Issue by foye501** [RasaHQ/retail-demo#32]
verified_email is a slot defined in the domain. I can't find it anywhere, how it works

**Comment by sara-tagger** [comment]
Thanks for the issue, @melindaloubser1 will get back to you about it soon!

###### You may find help in the [docs](https://rasa.com/docs/) and the [forum,](https://forum.rasa.com/) too 🤗

**Comment by foye501** [comment]
Thank you

**Comment by indam23** [comment]
You're right, this seems to be a leftover of a story that doesn't exist anymore. @mvielkind maybe you know if it can be removed?
